keps/sig-apps/0003-kubernetes-application-api.md

```diff
@@ -0,0 +1,1120 @@
+---
+kep-number: draft-20180115
+title: Kubernetes Application API
+authors:
+  - "@kow3ns"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-cli
+reviewers:
+  - "@kow3ns"
+  - "@danielromlein"
+  - "@lavalamp"
+approvers:
+  - "@kow3ns"
+  - "@mattfarina"
+  - "@prydonius"
+  - "@pwittrock"
+editor:
+creation-date: 2018-01-15
+last-updated: 2018-01-15
+status: draft
```

Comment:
This is now called `accepted` to mean that the owning SIG has decided to spend time looking at the topic.